x-pack/plugins/security_solution/public/resolver/view/panels/related_event_detail.tsx

```diff
@@ -90,6 +56,49 @@ const TitleHr = memo(() => {
 });
 TitleHr.displayName = 'TitleHR';
 
+const GeneratedText = React.memo(function ({ children }) {
```

Comment:
Referencing @kqualters-elastic comment from slack, I would cc a member of the EUI team to see about moving this logic there and getting that process started

Reply:
Yeah, I think that's a good idea for the long term, too. Seems like it would be useful in a lot of places. @chandlerprall has always been helpful, maybe he could give us an idea.

Reply:
for any context since I still have a terrible habit of saying too much there and not enough here: long term imo it might make sense to have this logic live in https://github.com/elastic/eui/tree/master/src/components/text as text_wrap.tsx and text_wrap.test.tsx, then used in the base text component conditionally based on the presence of a prop. Add a few snapshot tests that verify the presence of <wbr />'s where you expect them, add as many test cases as needed

Reply:
Feel free to open an issue in EUI for discussion! It's possible we have this solved via a css utility class.